### ISSUE TITLE
Improvement: Provide different landing page and ticket notes for "I need help, please contact/guide me" option during migration flow

### ISSUE THREAD

**Issue by cuemarie** [Automattic/wp-calypso#97664]
### TLDR

Clicking this link should lead to a "We'll be in touch" type of landing page instead, that does not promise we can/will migrate their site yet.  I think it should also not set the destination site into "migration started", and, ideally, the zendesk tickets this flow generates can be updated to be more accurate to where the customer asked for help in the process.

### Current Flow

During the migration flow, there are a couple points at which a customer can select `I need help, please contact me`; once on the "Tell us about your WordPress site" page, and also on the "Looks like there's been a mix-up" page, (`I need help, please guide me`) if a customer tries the flow with a non-wp site.

| Tell us about your WordPress Site | Looks like there's been a mix-up |
| ------------- | ------------- |
| ![Image](https://github.com/user-attachments/assets/26fb10c3-f829-4b18-8b55-08ab1c54d8db) | ![Image](https://github.com/user-attachments/assets/a017a042-e423-48a7-be71-523b76940d78) |

Clicking this link _on either page_ takes the user to the same "We'll take it from here!" page we show when they do submit a WP site and credentials, which advises that the customer is all done, and sets the expectation that we'll migrate their site in 2-3 business days.  

![Image](https://github.com/user-attachments/assets/3a214f4c-5993-45dd-98c8-153079842e23)

It also then changes their destination site into "Migration Started", and again sort of promises the user their migration is happening.  

![Image](https://github.com/user-attachments/assets/ad3233fb-e0f9-4e50-8754-fb634d2734c6)

---

Lastly, these flows generate a generic ticket in the migration queue, identical between the two flows noted above, with the following note:

`The user has requested an assisted migration in one of our migration flows. This ticket was created automatically to get the request into our ZenDesk migration queue. Next steps are to contact the user to get migrations details from them.`

Examples: 
1. Ticket from "Tell us about your WordPress site": 9196241-zen
2. Ticket from "Looks like there's been a mix-up": 9196268-zen

### Why

If a customer is submitting a site that is not a WordPress site, they should not be told that their migration is in process and that we'll be done in 2-3 business days, as this is not true.  They are about to get one of two possible emails from support: 

1. (If they have a WP site) details on what to expect, and a request for their credentials _in order to get started_; i.e., their migration is not underway until they reply.
2. (If they have a non-WP site) a reply stating we cannot migrate their site; which is likely to be frustrating after the flow just told them it was underway.

### How

For the customer's experience, a `We'll be in touch soon!` page would be much better; it could also reiterate that "If your site is not yet built with WordPress, then we can't migrate it for you, but here are some links and also DIFM instead".

For the Tickets in Zendesk, noting where they were in the flows when they asked for help would be helpful context for HEs. For example:

From the `Tell us about your WordPress site` page:

> The user has requested we contact them about a migration in lieu of sharing any WordPress site credentials or Backup files.  Please double check their submitted source site.

From the `Looks like there's been a mix-up` page:

> The user has requested we contact them about a migration, but the migration flow detected their source site is not built with WordPress.  Please double check their submitted source site.

**Comment by Robertght** [comment]
I'm not changing its type yet, but I agree it would be great to have and improve clarity and expectations, especially for newcomers.

I'm adding a High priority and flagging it for @Automattic/serenity

**Comment by cuemarie** [comment]
Thanks! I should have added that I also chatted about this with Serenity here: p1734625232781969-slack-C0Q664T29

**Comment by gabrielcaires** [comment]
@cuemarie The correct step page should be this:

![Image](https://github.com/user-attachments/ssets/7190f2ba-4dc4-4fe4-a41d-e77f7c277562)

The PR [Site Migration: Show correct landing page when the user asks for help #98092](https://github.com/Automattic/wp-calypso/pull/98092) is fixing it. 

Thanks for reporting it.
